src/com/keep/calm/enjoy/java/c/loop/and/array/parnia/assignmentc/GradeProgram.java

```diff
@@ -4,7 +4,7 @@ public class GradeProgram {
     public static void main(String[] args) {
 
             Scanner scanner = new Scanner(System.in);
-            System.out.println("Enter Grade of student");
+            System.out.println("Enter Grade of students");
```

Comment:
I am not sure what are these signs + and - at the beginning of the lines, but I think it is better to remove extra codes in order to have a cleaner code. Thx.

Reply:
+ and - added by github and + means this line has been added and - means this line has been removed